scripts/load_from_archive.py

```diff
@@ -31,7 +31,7 @@ def main():
     for date in date_range(START_DATE, END_DATE):  # this is inclusive of END_DATE
         for path in [
             ["input"],
-            ["mcip"],
+            ["cmaq"],
```

Comment:
I guess we try to sync `cmaq` data if it was already run, but if `SKIP_CMAQ_SETUP` is **not** specified, might we still need `mcip`?

Reply:
The cmaq data from the daily runs should always be available (if they ran successfully). The SKIP_CMAQ_SETUP should only be ran of the monthly workflow. All of this is an optimisation because of how we split the workflows.

This script has a preface mentioning that is meant to be run via the mechanisms outside of this repo.

Reply:
Makes sense!